diff -ruN orig/src/audio_thread_alsa.c new/src/audio_thread_alsa.c
--- orig/src/audio_thread_alsa.c	2019-05-24 05:33:09.000000000 -0600
+++ new/src/audio_thread_alsa.c	2019-05-25 23:35:22.534932285 -0600
@@ -29,7 +29,7 @@
 #include "trace_core.h"
 #include "trace_helpers.h"
 #include "utils.h"
-#include <asoundlib.h>
+#include <sys/asoundlib.h>
 #include <errno.h>
 #include <glib.h>
 #include <poll.h>
